skyfield/documentation/stars.rst

```diff
@@ -107,12 +107,15 @@ Here are all three effects applied to Barnard’s star:
 
 .. testcode::
 
+    from skyfield.api import T0
+
     barnard = Star(ra_hours=(17, 57, 48.49803),
                    dec_degrees=(4, 41, 36.2072),
                    ra_mas_per_year=-798.71,
                    dec_mas_per_year=+10337.77,
                    parallax_mas=545.4,
-                   radial_km_per_s=-110.6)
+                   radial_km_per_s=-110.6,
+                   jd_of_position=T0)
```

Comment:
I suspect we should expand this to two separate examples, one without, and one with the new keyword, since many users cut-and-paste without reading the text. But I can do that after this lands if you don't get around to it!

Reply:
As per Twitter conversion from yesterday and today, I suspect? I mostly avoided this since an example that was *almost* entirely redundant seemed a bit lame, but I didn't want to go to the work of coming up with and validating another example :-)